apps/sertra/components/Navbar.tsx

```diff
@@ -1,11 +1,15 @@
 import Link from "next/link";
 import { FC, PropsWithChildren } from "react";
 
+import Logo from "/public/ts-logo.svg";
 import styles from "./Navbar.module.css";
 
 export const Navbar: FC<PropsWithChildren> = () => (
   <nav className={styles.navbar}>
-    <h1>Thunderstore</h1>
+    <h1>
+      <Logo alt="" />
```

Comment:
So SVG has no accessibility feature similar to image alt text?

Reply:
In case you'd need a textual clarification for inline SVGs, you can use `<title>` inside `<svg>`. That being said, by large it's not needed, even less so with purely decorative elements.